e2e-tests/gef/cypress/e2e/scenarios/eligibility-criterion-21/eligibility-criterion-21.spec.js

```diff
@@ -0,0 +1,95 @@
+import relative from '../../relativeURL';
+import automaticCover from '../../pages/automatic-cover';
+import ineligibleAutomaticCover from '../../pages/ineligible-automatic-cover';
+import manualInclusion from '../../pages/manual-inclusion-questionnaire';
+import { BANK1_MAKER1 } from '../../../../../e2e-fixtures/portal-users.fixture';
+
+let dealId;
+
+context('Eligibility Criterion 21', () => {
+  before(() => {
+    cy.loadData();
+    cy.apiLogin(BANK1_MAKER1)
+      .then((token) => token)
+      .then((token) => {
+        cy.apiFetchAllApplications(token);
+      })
+      .then(({ body }) => {
+        dealId = body.items[0]._id;
+      });
+    cy.login(BANK1_MAKER1);
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+    cy.visit(relative(`/gef/application-details/${dealId}/automatic-cover`));
+  });
+
+  describe('Visiting page eligibility criteria page', () => {
+    it('displays the correct elements', () => {
+      automaticCover.mainHeading();
+      automaticCover.form();
+      automaticCover.automaticCoverTerm(21).its('length').should('be.gt', 0); // contains terms
```

Comment:
```suggestion
      automaticCover.automaticCoverTerm(21).should('exist');
```
I think this test is just trying to check that EC 21 is shown